app/controllers/application_controller.rb

```diff
@@ -1,30 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
+  before_action :set_current_player
 
-  def chart_data(game)
```

Comment:
moved to `games_controller` as it's the only place this method gets called.   
made some minor changes to the code as well: 
`Set` isn't adding any value and it takes an extra `to_a`, so changed to `Array`